checker/src/main/java/org/checkerframework/checker/units/UnitsRelations.java

```diff
@@ -34,4 +37,51 @@
      *     is known
      */
     @Nullable AnnotationMirror division(AnnotatedTypeMirror lht, AnnotatedTypeMirror rht);
+
+    /**
```

Comment:
What is the motivation for moving these methods here?
Similarly-named methods remain in `UnitsRelationsTools`.
It would seem cleaner to me to keep all these related methods in `UnitsRelationsTools` and not add them as static methods to this interface.

Reply:
The other version of the methods are the desirable Name ones.
These are the undesirable Class ones, which should not be used by most clients.
These are needed as a convenience for UnitsRelations, but are here so they are not noticeable to other clients.

Reply:
But the UnitsRelations interface is the more visible place, because one would have to implement that interface for custom units.
How about moving the methods back and adding javadoc when to use them?

Reply:
When implementing custom units, using the class versions is acceptable.  Otherwise, it isn't.  So this moves the class version to a place accessible to the people who need it (and no one else).
It keeps them out of UnitsRelationsTools, which is a class that more clients such as the type factory use.  Keeping them less visible to the type factory reduces the likelihood that they will be used incorrectly, compared to Javadoc.
Also note that one of the classes contains 3 methods and the other contains 2 methods, a difference that would also have to be documented if they are all put together.

But, if you feel strongly about this, I can move it back.